recipe/meta.yaml

```diff
@@ -10,11 +10,12 @@ source:
   sha256: 70da7a693ff7a6a283850ab6d62acf628eea17d386488af8918576d0760aef7b
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
```

Comment:
i see. ok i was confused about that.

Reply:
Yeah, it's more accurate (though still not 100%) to say that `{{ stdlib("c") }}` is needed everywhere something gets compiled, rather than it's just for C